Models/INSERT.ts

```diff
@@ -0,0 +1,20 @@
+import { Request , Response } from "express";
+import { StudentProps } from "../Interfaces/studentinterface";
+import { executeQuery } from "../utils/database";
+import { hashPassword } from "../Helpers/utility";
+import { SELECTSTUDENTQUERY } from "./SELECT";
+
+
+export const creationOfStudent = async (req: Request , res: Response) : Promise<void> => {
+    try {
+        const INSERTSTUDENTQUERY = "INSERT INTO `Student` (StudentFname , StudentLname , StudentDoB  , StudentMajor) VALUES (?,?,?,?)" ;
+        const userPassInput = req.params.password;
```

Comment:
inside models we getting the body of request?

Reply:
Done